bin/makepart

```diff
@@ -16,10 +16,9 @@ mkdir -p "$rootfs_work/overlay"/{etc,var}/{upper,work}
 
 find "$rootfs_work/var/log/" -type f -delete
 
+# SELinux 'default' policy file contexts
 chcon -R system_u:object_r:unlabeled_t:s0 "$rootfs_work"
-# Temporary deactivated until this is fixed by upstream:
-# GitHub: https://github.com/gardenlinux/gardenlinux/issues/1014
-#chroot "$rootfs_work" /usr/bin/env -i /sbin/setfiles /etc/selinux/default/contexts/files/file_contexts /
+/opt/gardenlinux/bin/garden-chroot "$rootfs_work" /sbin/setfiles /etc/selinux/default/contexts/files/file_contexts /
```

Comment:
A small design decision may be hiding here. I think this is fine, but want to hear your perspective before approving. A short reply is also fine for me.

Question: Can we safely assume that `/opt/gardenlinux/bin/` programs exist when `bin/makepart` is called?

In theory, it would sound nice to not introduce a hard coded path to its dependencies in /opt/gardenlinux.
 
However, this discussion would be bikeshedding - so I lean towards accepting that assumption. And just fix the path when we need to.

As long as [build.sh](https://github.com/gardenlinux/gardenlinux/blob/3f5bc0a235285ccbb3add8980730e1a29dbd5fff/build.sh#L121) does it that way, we should be fine.

Reply:
I agree with @Vincinator, we should probably not hard code the path.
Why not using the variable `thisDir` like in other scripts within the `bin/` directory? This should be sufficient, no?
```
thisDir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
```

Then, we could use the variable it like this:
```
"$thisDir/garden-chroot" "$rootfs_work" /sbin/setfiles /etc/selinux/default/contexts/files/file_contexts /
``` 

What do you think?

Reply:
Thank you both for the input. I think _thisDir_ is perfectly fine for this (and intended for this cases).